### ISSUE TITLE
Event class is lost when scrolling off screen

### ISSUE THREAD

**Issue by harohl** [fullcalendar/fullcalendar-scheduler#229]
When using event.className and setting a font and background color, this works just fine.  However, when using event.className to set a text-decoration to set line-through, the line-through is only visible when the arrow (indicating that the event continues off the view) is not present.  Below is my class definition, and the attached files have print-screens of the event with the line-through and when the line-through is missing - only due to scrolling off the view.

![line-through1](https://cloud.githubusercontent.com/assets/19411389/19158184/d5168588-8bb5-11e6-84ca-60b53cc798b3.jpg)
![line-through2](https://cloud.githubusercontent.com/assets/19411389/19158185/d51e3ec2-8bb5-11e6-9b6a-37298081231d.jpg)

`.ovly {
            text-decoration: line-through !important;
        }`

**Comment by arshaw** [comment]
This is a legitimate bug on fullcalendar-scheduler's end. It utilizes an optimization technique to make those names smooth as you scroll.

The solution would entail adding `text-decoration` to this list int the source code:
https://github.com/fullcalendar/fullcalendar-scheduler/blob/master/src/util/ScrollFollowerSprite.coffee#L175

I will get to this as soon as I can.

**Comment by harohl** [comment]
Can you give a rough timespan of when this will be added to the source code?  I need to decide on whether to change the design or wait for the code.  Or, are you suggesting I can change my version of the source code?

**Comment by harohl** [comment]
Also, in case I add other properties to the class, are you going to make sure all modifications to the style are kept when scrolling?  We are using various styles to indicate different things at a glance of the schedule.

**Comment by arshaw** [comment]
It's a pretty easy fix from my end, so I'll get it into the next release, which will be within the next two weeks.

If you need a fix in the meantime, it wouldn't be too hard for you to modify it. You could either modify the source CoffeeScript code and compile, or you could probably easily find the corresponding lines in the build JS and modify it there.

**Comment by arshaw** [comment]
What other properties do you need that might neglected to be cloned @harohl ?

**Comment by harohl** [comment]
I am still working on the design for all of the ways we will show different statuses of the schedules.  The properties we will be considering are anything that will differentiate schedules from one another.  So, after a quick look at CSS properties, these might include font properties (e.g. style, weight, color), text properties (decoration), background (color, image?, repeat?), border (style, width, color).  I am also not sure if there is anything in animation that we would want to use for flashing, etc.  You might have a better idea than me about what properties make sense for differentiating schedule properties at a glance.  Any suggestions on best properties to consider would be welcome.

Heidi

**Comment by arshaw** [comment]
here's the list of currently support properties:

```
position
z-index
font-weight
font-size
font-family
color
padding-top
padding-bottom
padding-left
padding-right
```

I'll add `text-decoration`. Please let me know if you discover you need any others.

**Comment by harohl** [comment]
I would like border style, width, and color.  These already seem to be supported though.  Am I right?

**Comment by arshaw** [comment]
yes, those are already supported. I guess I'm mainly just hoping to include all *text-related* CSS properties. I think we have them all accounted for. thanks.

**Comment by arshaw** [comment]
implemented here: https://github.com/fullcalendar/fullcalendar-scheduler/commit/f82758499f9479ca74cf5a7522ed62585b0586b1

will be released very soon

**Comment by arshaw** [comment]
fixed and released in [v1.5.0](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.5.0)

example: http://jsbin.com/torekoduwi/1/edit?css,output
